df.ui-menus.xml

```diff
@@ -288,16 +288,37 @@
             <pointer type-name='entity_position'/>
             <pointer type-name='entity_position_assignment'/>
             <pointer type-name='entity_position'/>
-            <int8_t name='in_squad'/>
-            <int8_t/>
-            <int8_t/>
-            <int8_t/>
-            <int32_t/>
-
-            <int32_t name="unk_80"/>
-            <int32_t name="unk_84"/>
-            not sure if vector is above or below the two ints
+            <bool name='unk3'/>
+            <bool name='unk4'/>
+            <bool name='squad_list_opened'/>
```

Comment:
Can you keep the `in_squad` name here?

Reply:
I cleared its name because I couldn't figure out if it actually means in_squad. It seems to be set for units that are not in_squad. Also it is set before squad list is opened. So I couldn't figure out what that bool actually was supposed to signal. But same time I noticed that different bool was indicating if squad list was opened after opening the unitview.

Reply:
Ok, maybe it moved, then. Thanks.

Reply:
Also in_squad is not used anywhere.